atlp-rwanda/elite-bn-fe
# ch(set-up-redux): set up redux with toolkit

What does this PR do?
Set up react redux with toolkit and persist 
Redux is JavaScript libraries, which is used for managing and updating application state. 
While Redux persist is  a tool used to seamlessly save the application's Redux state object to AsyncStorage. 

How should this be manually tested?
pre-requisite install node and yarn make sure you have the updated versions.

Initial clone the repo

Run : `yarn install`

After running the command `yarn start` it will direct you to the landing page, 
          The landing page contain nav bar with links to login, home, and counter 
          click on counter you can see an example where you can increase and decrease number. 

To test redux persist refresh the page and the previous number will be the same.

Any background context you want to provide?
What are the relevant pivotal tracker stories?
[#182201516](https://www.pivotaltracker.com/story/show/182201516)
